tests/providers/amazon/aws/hooks/test_sqs.py

```diff
@@ -28,8 +28,8 @@
 
 
 @unittest.skipIf(mock_sqs is None, 'moto sqs package missing')
-class TestAwsSQSHook(unittest.TestCase):
+class TestAwsSqsHook(unittest.TestCase):
```

Comment:
Should this be changed to `TestSqsHook`?

Reply:
changed